default/AI/ProductionAI.py

```diff
@@ -62,7 +63,14 @@ def find_best_designs_this_turn():
         ShipDesignAI.Cache.print_best_designs()
 
 
-def get_design_cost(cur_turn, design, pid):
+def get_design_cost(design, pid):  # TODO: Use new framework
+    """Find and return the design_cost of the specified design on the specified planet.
+
+    :param design:
+    :param pid: int: planet id
```

Comment:
Try next in pycharm:
- hover mouse on function usage and wait for tooltip
- press `CTRL+Q` (Quick doc action)
- replace docstring to:
```python
:param design: design
:type design: ShipDesignAI.ShipDesigner
:param pid: planet id
:type pid: int
```
- repeat inspections